client/service/client_service.go

```diff
@@ -0,0 +1,115 @@
+package service
+
+import (
+	"crypto/x509"
+	"fmt"
+
+	"github.com/scalar-labs/scalardl-go-client-sdk/v3/client/config"
+	"github.com/scalar-labs/scalardl-go-client-sdk/v3/crypto"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+// ClientService defines the interface of the client service.
+type ClientService struct {
+	clientConfig                config.ClientConfig
+	signer                      crypto.Signer
+	ledgerConnection            *grpc.ClientConn
+	ledgerPrivilegedConnection  *grpc.ClientConn
+	auditorConnection           *grpc.ClientConn
+	auditorPrivilegedConnection *grpc.ClientConn
+}
```

Comment:
This is the main structure.
All functions that implement the client interfaces are associated with this structure.